akka-http/src/main/resources/reference.conf

```diff
@@ -39,5 +39,5 @@ akka.http.routing {
 
   # Fully qualified config path which holds the dispatcher configuration
   # to be used by FlowMaterialiser when creating Actors for IO operations.
-  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher}
+  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher} // TODO point to akka.stream.file-io-dispatcher instead?
```

Comment:
+1 and shouldn't we create an issue for changing the akka.io.tcp.file-io-dispatcher to point to a dedicated io dispatcher?